src/main/java/seedu/address/logic/parser/AddClientCommandParser.java

```diff
@@ -42,9 +45,15 @@ public AddClientCommand parse(String args) throws ParseException {
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+        Birthday birthday = null;
```

Comment:
I feel that `Optional` should be implemented here before it is being passed on to `client`.
Rationale being handle it asap to prevent `NullPointerException` as much as possible